lib/commands/serve.js

```diff
@@ -67,12 +69,19 @@ class ServeCommand extends Command {
         }
       }
 
-      const pathname = urls
-        .map(url =>
-          `${url.replace(/^\//, '').replace(/\?.*/, '')}.js`.replace(
-            '.js.js',
-            '.js'
-          )
+      let pathname = urls
+        .map(
+          url =>
```

Comment:
this can probably be simplified, but in short, we only match Typescript lambdas if they end with `.ts` explicitly, when requested. Either manually or via a routes alias in now.json.